.rustfmt.toml

```diff
@@ -1,3 +0,0 @@
-# Temporarily disable rustfmt completely to avoid conflicts of newly formatted
-# code with old PRs.
-ignore = ["/"]
```

Comment:
If we really enable `rustfmt`, I'd prefer deleting this file in a separate commit. (So that the commit could easily be undone without affecting anything else.)